rust_snuba/src/processors/generic_metrics.rs

```diff
@@ -65,6 +65,7 @@ struct FromGenericMetricsMessage {
     #[serde(flatten)]
     value: MetricValue,
     retention_days: u16,
+    sampling_weight: Option<u64>,
```

Comment:
This can be a float or int. Can we make the required change to handle both the cases?

Reply:
Yes, it's a floating point now in the `FromGenericMetricsMessage` I'm casting it into a u64 during parsing